autopush/manifest.json

```diff
@@ -1,35 +1,34 @@
 {
-    "envs" : {
-	"stage": {
-	    "test_types" : [ "stack-check" ],
-            "aws_region": "us-east-1",
-            "stack_label" : "autopush-stage",
-            "aws_region": "us-east-1",
```

Comment:
Added JSON linting via ESLint [**eslint-plugin-json**](https://github.com/azeemba/eslint-plugin-json) module, and it reported this:
```
/Users/pdehaan/dev/tmp/del/services-test/autopush/manifest.json
  7:7  error  Duplicate key 'aws_region'  bad-json
```